onnx/backend/test/case/node/lstm.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class LSTM(Base):
+
+    @staticmethod
+    def export_defaults():
+        input = np.array([[[1., 2.], [3., 4.], [5., 6.]]]).astype(np.float32)
+
+        input_size = 2
+        hidden_size = 4
+
+        node = onnx.helper.make_node(
+            'LSTM',
+            inputs=['X', 'W', 'R'],
+            outputs=['Y'],
+            hidden_size=hidden_size
+        )
+
+        W = np.ones((1, 4 * hidden_size, input_size)).astype(np.float32)
+        R = np.ones((1, 4 * hidden_size, hidden_size)).astype(np.float32)
+
+        output = np.array([[0.70377535, 0.70377535, 0.70377535, 0.70377535],
+                           [0.96009213, 0.96009213, 0.96009213, 0.96009213],
+                           [0.99451119, 0.99451119, 0.99451119, 0.99451119]]).astype(np.float32)
+
+        expect(node, inputs=[input, W, R], outputs=[output], name='test_lstm_defaults')
+
+    @staticmethod
```

Comment:
Can we add tests for bidirectional LSTMs please?

Reply:
The scope of this PR is to provide basic test coverage for forward case, as well as reference numpy implmentation for it. Bidirectional LSTMs shall be added seperately as part of other PRs.